Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:testing
+RUN apt-get update && apt-get -y install git python-pip python-dev
+RUN cd ; git clone https://github.com/komitywa/wysadzulice.pl.git
+# TODO: factor out code from .travis.yml and here to a separate script.
+RUN cd ~/wysadzulice.pl && pip install -r requirements.txt
+RUN cd ~/wysadzulice.pl && pep8 --exclude='wysadzulice/migrations/*' .
```

Comment:
to powinno trafić do `docker/pep8` lub do `docker/tests` lub do `docker/linters`. jako coś, czego nie wykonujemy podczas buildów.